BabyPatterns/FeedingsVM.swift

```diff
@@ -1,11 +1,74 @@
 import Foundation
 import Framework_BabyPatterns
 import Library
+import WatchConnectivity
 
-class FeedingsVM: Loggable {
+extension FeedingsVM: WCSessionDelegate {
+    public func session(_ session: WCSession, activationDidCompleteWith activationState: WCSessionActivationState, error: Error?) {
+        print("Completed activation: \(error?.localizedDescription ?? "n/a error")")
+
+    }
+    public func sessionDidBecomeInactive(_ session: WCSession) {
+        print("session did become inactive")
+    }
+
+    public func sessionDidDeactivate(_ session: WCSession) {
+        print("session did deactivate")
+    }
+    public func session(_ session: WCSession, didReceiveMessage message: [String : Any]) {
+        print("Received message!")
+    }
+    public func session(_ session: WCSession, didReceiveMessage message: [String : Any], replyHandler: @escaping ([String : Any]) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c49e444d61917df61aa42d9cccb11a46857e6fb9/BabyPatterns/FeedingsVM.swift#L21"></span>Line should be 120 characters or less: currently 140 characters<br />
<code>line_length</code> <code>FeedingsVM.swift:21</code></td>
    </tr>
  </tbody>
</table>